### PR TITLE
Format time display and edit cards

### LINKED ISSUE #107: Format time display
Display in 12 hour format

### PR DESCRIPTION
**Pull request by KnytJC**
close #107

### CODE REVIEW

File: src/main/java/seedu/address/model/task/TaskDateTimeFormatter.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package seedu.address.model.task;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
```
Comment (kabir-plod) [inline src/main/java/seedu/address/model/task/TaskDateTimeFormatter.java]: //@@ author tag missing :)

File: src/main/java/seedu/address/ui/EventTaskCard.java [hunk 2]
```diff
@@ -50,12 +49,21 @@ public void initialize() {
...
         id.setText(displayedIndex + ". ");
         taskType.setText(task.getTaskType().toString());
-        startDateAndTime.setText(task.getStartDate().get().toString());
-        endDateAndTime.setText(task.getEndDate().get().toString());
+        setTaskDateTime();
         setTaskStatus();
         tags.setText(task.tagsString());
     }
 
+	private void setTaskDateTime() {
```
Comment (kabir-plod) [inline src/main/java/seedu/address/ui/EventTaskCard.java]: @@author tag

File: src/main/java/seedu/address/model/task/TaskDateTimeFormatter.java (hunk 1 above)
Comment (kabir-plod) [inline src/main/java/seedu/address/model/task/TaskDateTimeFormatter.java]: //@@author missing :)

File: src/main/java/seedu/address/ui/EventTaskCard.java (hunk 2 above)
Comment (kabir-plod) [inline src/main/java/seedu/address/ui/EventTaskCard.java]: //@@author. Consider review to be done after making these changes

**Comment by kelvinnpc** [comment]
reviewed

File: src/main/java/seedu/address/ui/EventTaskCard.java (hunk 2 above)
Comment (KnytJC) [inline src/main/java/seedu/address/ui/EventTaskCard.java]: okok  thanks lol

**Pull request merged**